charts/aiven-operator/values.yaml

```diff
@@ -6,6 +6,10 @@ replicaCount: 1
 
 nameOverride: ""
 fullnameOverride: ""
+defaultToken: ""
+metricsBindAddress: ""
```

Comment:
defaults seem broken here; those values will probably lead to a broken command being executed right?

Reply:
nope, in case it is empty these feature gets disabled